docs_nnx/nnx_basics.ipynb

```diff
@@ -384,21 +384,21 @@
    "source": [
     "There are two things happening in this example that are worth mentioning:\n",
     "\n",
-    "1. The updates to each of the [`nnx.BatchNorm`](https://flax.readthedocs.io/en/latest/api_reference/flax.nnx/nn/normalization.html#flax.nnx.BatchNorm) and [`nnx.Dropout`](https://flax.readthedocs.io/en/latest/api_reference/flax.nnx/nn/stochastic.html#flax.nnx.Dropout) layer's state is automatically propagated from within `loss_fn` to `train_step` all the way to the `model` reference outside.\n",
-    "2. The `optimizer` holds a mutable reference to the `model` - this relationship is preserved inside the `train_step` function making it possible to update the model's parameters using the optimizer alone.\n",
+    "1. The updates to each of the `BatchNorm` and `Dropout` layer's state is automatically propagated from within `loss_fn` to `train_step` all the way to the `model` reference outside.\n",
```

Comment:
Outside of the smaller NNX community, external users would need to know if BatchNorm / Dropout is an Flax NNX or flax.nn (Linen) BatchNorm. We should at least specify this here (entire doc). cc @IvyZX

Reply:
@jburnim WDYT